File: Week6/sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

def main():
    height = get_height()
    blanks = height - 1
    hash = 1
    for i in range(height):
        for k in range(blanks):
            print(" ", end="")
        for l in range(hash):
            print("#", end="")
        print("  ", end="")
        for m in range(hash):
            print("#", end="")
        print()
        blanks -= 1
        hash += 1



def get_height():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            return height

main()

File: Week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int x = 0;

    while(x < 1 || x > 8)
    {
        x = get_int("Size: ");
    }

    int y = 1;
    for(int j = 0; j < x; j++)
    {
        int z = x - y;

        for(int k = 0; k < z; k++)
        {
            printf(" ");
        }
        for(int l = 0; l < y; l++)
        {
            printf("#");
        }
        printf("  ");
        for(int m = 0; m < y; m++)
        {
            printf("#");
        }

        printf("\n");
        y++;

    }
}